src/geo/cartodb-layer-group.js

```diff
@@ -19,6 +19,16 @@ var CartoDBLayerGroup = Backbone.Model.extend({
     this._layersCollection = options.layersCollection;
   },
 
+  addError: function (error) {
```

Comment:
LGTM! We can always change this if we need to "store" the error or if we need to be notified when the error is gone.